lib/motion-spec/mock/method_bind_unbind.rb

```diff
@@ -0,0 +1,57 @@
+# -*- encoding : utf-8 -*-
+# The meta-programming that allows us to pop methods on and off for mocking
+class Object
+  # The hidden singleton lurks behind everyone
+  def metaclass
+    class << self
+      self
+    end
+  end
+
+  def meta_eval(&block)
+    metaclass.instance_eval &block
```

Comment:
Ambiguous block operator. Parenthesize the method arguments if it's surely a block operator, or add a whitespace to the right of the `&` if it should be a binary AND.